repos/spack_repo/builtin/packages/py_bidict/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyBidict(PythonPackage):
+    """The bidirectional mapping library for Python."""
+
+    homepage = "https://github.com/jab/bidict"
+    pypi = "bidict/bidict-0.23.1.tar.gz"
+
+    license("MPL-2.0", checked_by="github_user1")
```

Comment:
```suggestion
    license("MPL-2.0")
```

Accidental placeholder left in package. Suggest to remove or add yourself.

Reply:
Sorry that was indeed an overlook on. Thank you very much.